code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -266,7 +266,7 @@
 					CHECK_TICK
 
 /obj/structure/destructible/clockwork/massive/celestial_gateway/attack_ghost(mob/user)
-	if(!user.client || !user.client.holder)
+	if(!user.client || !user.client.holder || !IsAdminGhost(user))
```

Comment:
`!user.client.holder` does this

Reply:
ITS A FUCKING RIPPIE ISSUE, NOT A /TG/ ISSUE

Reply:
just fucking replace the whole thing with IsAdminGhost(user), see https://github.com/tgstation/tgstation/blob/626302c1bf4bb2a37aa6508145751f6c4c78696d/code/modules/mob/mob_helpers.dm#L432-L443

Reply:
check_rights_for is defined in https://github.com/tgstation/tgstation/blob/d3dcc118ecb5f5410de684b1bb46289f8066ab23/code/modules/admin/holder2.dm#L130-L136